.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          # - '1.6'
+          - '1'
```

Comment:
This is latest, not LTS, so is inconsistent with the PR title / commit message. Not sure what the intention was.

Reply:
Hmm definitely not consistent with the intention. But I will change the title for now.